Application/EdFi.Ods.Api/Security/AuthorizationStrategies/CustomViewBased/CustomViewBasedAuthorizationStrategyProvider.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using Autofac.Extras.DynamicProxy;
+using EdFi.Ods.Common.Caching;
+using EdFi.Ods.Common.Models.Domain;
+using EdFi.Ods.Common.Security.Authorization;
+using log4net;
+
+namespace EdFi.Ods.Api.Security.AuthorizationStrategies.CustomViewBased;
+
+[Intercept(InterceptorCacheKeys.Security)]
```

Comment:
This provider is cached with the security metadata, getting cleared when the security metadata is cleared.